plugins/opensearch/loggable-transport-netty4/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
It seems like almost all of the other files were moved _out_ of `plugins/opensearch/loggable-transport...` and into `plugins/elasticsearch/loggable-transport...`. Why are these two files added here?